_publications/2022-04-11-portmaster-vs-simplewall.md

```diff
@@ -183,30 +171,21 @@ FIXME - Feedback:
 Simplewall only shows you IPs - not domains. Hard to see what is actually going on!
 -->
 
-### Simplewall Is [Just] A Graphical User Interface For the Windows Filtering Platform @Daniel
+### Simplewall Is Basically A Graphical User Interface for the Windows Filtering Platform
 
-Simplewall uses the Windows Filtering Platform (WFP) in user-space mode, creating rules for applications essentially exactly like the Windows Firewall does itself. Simplewall is notified of new network activity through events. This means that Simplewall is limited to the data that Windows provides it with - which is mostly metadata about the packest, but it cannot see that data in the network packets itself.
+Simplewall uses the Windows Filtering Platform (WFP) in user-space mode, creating rules for applications essentially exactly like the Windows Firewall does itself. Simplewall is notified of new network activity through events. This means that Simplewall is limited to the data that Windows provides it with. Simplewall can mostly see metadata about the packets, but it cannot see the data in the network packets itself.
```

Comment:
Kill "mostly", it don't feel it make sense in the new sentence structure anymore.